Add cgroup memory support in kernel build config, and command line to…

davechapman442 opened the pull request:
… enable

Otherwise systemd cannot mount to sys fs, and cthulhu start up fails - meaning all LCM start up fails

On branch 53-bug-raspberry-pi-r4-build-does-not-enable-cgroup-v1-memory-controls-required-by-cthulhu-startup Changes to be committed:
	modified:   recipes-kernel/linux/files/dobby.cfg
	new file:   recipes-kernel/linux/rpi-cmdline.bbappend

davechapman442 committed:
Add cgroup memory support in kernel build config, and command line to enable
Otherwise systemd cannot mount to sys fs, and cthulhu start up fails - meaning all LCM start up fails

On branch 53-bug-raspberry-pi-r4-build-does-not-enable-cgroup-v1-memory-controls-required-by-cthulhu-startup
Changes to be committed:
	modified:   recipes-kernel/linux/files/dobby.cfg
	new file:   recipes-kernel/linux/rpi-cmdline.bbappend

piotr-nakraszewicz-red commented on recipes-kernel/linux/files/containers.cfg:
Due to a problem in cthulhu a fix is provided in dobby.cfg.
It doesn't make sense semantically. Can dobby.cfg be renamed to a more generic name, that would clearly state that it is container related and not only dobby related?

davechapman442 committed:
-76 Dobby poorly named

Renamed dobby.cfg to containers.cfg, and reference in bbappend

On branch 76-bug-dobbycfg-is-poorly-named-should-be-more-generic-eg-containerscfg
Changes to be committed:
	renamed:    recipes-kernel/linux/files/dobby.cfg -> recipes-kernel/linux/files/containers.cfg
	modified:   recipes-kernel/linux/linux-%.bbappend

davechapman442 committed:
Merge branch 'main' into 53-bug-raspberry-pi-r4-build-does-not-enable-cgroup-v1-memory-controls-required-by-cthulhu-startup

davechapman442 committed:
Merge branch '76-bug-dobbycfg-is-poorly-named-should-be-more-generic-eg-containerscfg' into 53-bug-raspberry-pi-r4-build-does-not-enable-cgroup-v1-memory-controls-required-by-cthulhu-startup

davechapman442 committed:
Merge branch 'main' into 53-bug-raspberry-pi-r4-build-does-not-enable-cgroup-v1-memory-controls-required-by-cthulhu-startup
